### ISSUE TITLE
Next-line-rules not detected after /* eslint-disable */

### ISSUE THREAD

**Issue by F1LT3R** [sindresorhus/eslint-plugin-unicorn#168]
I am using AVA with XO in SublimeText3, and I'm running into a weird situation. My best guess is that this behavior is coming from the `eslint-plugin-unicorn` plugin. I say that because SublimeText appears to handle the rules correctly, but running `xo` from the CLI gives me unexpected results.

I expect that using `// eslint-disable-next-line rule1 rule2` after using `/* eslint-disable */` and `/* eslint-enable */` should not show any errors on line 8.

<img width="684" alt="screen shot 2018-03-27 at 10 09 58 am" src="https://user-images.githubusercontent.com/1218446/37972853-3ddbd544-31a7-11e8-8b53-f88857c7849f.png">

So far, so good. I think. (Although the error over the `i` of `import` tells me something is going wrong, especially as I get no error message when hovering there as I do with all the other highlighted errors.)

Let's see what happens when I run `xo` from the CLI:

<img width="785" alt="screen shot 2018-03-27 at 10 15 05 am" src="https://user-images.githubusercontent.com/1218446/37973098-c2fef260-31a7-11e8-90a0-23aa60a38686.png">

It appears XO disagrees with SublimeText.

What happens if I remove the comments: `/* eslint-disable */` and `/* eslint-enable */`?

Now my code looks like this...

<img width="688" alt="screen shot 2018-03-27 at 10 16 17 am" src="https://user-images.githubusercontent.com/1218446/37973185-eef42f16-31a7-11e8-961b-1d254e4b782f.png">

What does XO say now?

<img width="376" alt="screen shot 2018-03-27 at 10 18 03 am" src="https://user-images.githubusercontent.com/1218446/37973299-2df9d3dc-31a8-11e8-9693-b721412d2137.png">

... nothing to report.

Is it me, or is this really weird? (Apologies in advance if I'm using the tools incorrectly.)

Here is the code in case you need to copy/paste for a test.

```js
import test from 'ava'

test('Should re-align indentation at multiple levels', t => {
	/* eslint-disable */
	/* eslint-enable */

	const obj = {
		// eslint-disable-next-line object-shorthand, no-unused-vars
		'baz-qux': function (a, b) {
			return 'Wombat!'
		}
	}

	t.true(typeof obj === 'object')
})
```

**Comment by F1LT3R** [comment]
🤦‍♂️ I just realized the `Specify the rules you want to disable at line 123:45` refers to the `/* eslint-disable */` comment, not the `// eslint-disable-next-line` comment.

The fact that the line numbers are wrong is what threw me I think. 

It's also weird that the error is not being highlighted on the correct line in Sublime.

Now I'm not so sure this bug is related to this package.

Thoughts?

**Comment by sindresorhus** [comment]
See https://github.com/sindresorhus/eslint-plugin-unicorn/blob/985a861382679ab42a723720634f064a5eb104a6/rules/no-abusive-eslint-disable.js#L16-L17 for why it's warning at line 0.
